ts/a11y/explorer/Region.ts

```diff
@@ -367,6 +367,273 @@ export class LiveRegion extends StringRegion {
 }
 
 
+const ProsodyKeys = [ 'pitch', 'rate', 'volume' ];
+
+interface ProsodyElement {
+  [propName: string]: string | boolean | number;
+  pitch?: number;
+  rate?: number;
+  volume?: number;
+}
+
+interface SsmlElement extends ProsodyElement {
+  [propName: string]: string | boolean | number;
+  pause?: string;
+  text?: string;
+  mark?: string;
+  character?: boolean;
+  kind?: string;
+}
+
+/**
+ * Region class that enables auto voicing of content via SSML markup.
+ */
+export class SpeechRegion extends LiveRegion {
+
+  /**
+   * Flag to activate auto voicing.
+   */
+  public active: boolean = false;
+
+  /**
+   * The math expression that is currently explored. Other regions do not need
+   * this node as the explorer administers both node and region, while only
+   * pushing output into the region. But in the case autovoicing the speech
+   * regions needs to mark elements in the node directly.
+   */
+  public node: Element = null;
+
+  /**
+   * Flag to indicate if a node is marked as being spoken.
+   */
+  private clear: boolean = false;
+
+  /**
+   * The highlighter to use.
+   */
+  public highlighter: Sre.highlighter = Sre.getHighlighter(
+    {color: 'red'}, {color: 'black'},
+    {renderer: this.document.outputJax.name, browser: 'v3'}
+  );
+
+  /**
+   * @override
+   */
+  public Clear(): void {
+    super.Clear();
+  }
```

Comment:
I'm not sure why this is being overridden since it only does the super action anyway.

Reply:
Probably did something during experimental phase. Removed now.